protocol/synthetix/contracts/storage/Pool.sol

```diff
@@ -194,44 +194,54 @@ library Pool {
     }
 
     /**
-     * @dev Implements a system-wide fail safe to prevent a market from taking too much debt.
+     * @dev Determines the resulting maximum value per share for a market, according to a system-wide minimum liquidity ratio. This prevents markets from taking too much debt.
```

Comment:
this prevents markets from assigning more debt to pools than they have collateral to cover

would be a better way to say it